.env

```diff
@@ -13,21 +13,27 @@ POSTGRES_PORT=5432
 # RM Services
 CASE_HOST=casesvc
 CASE_PORT=8171
+CASE_DEBUG_PORT=5171
```

Comment:
I wonder if we need any of these in the .env file or would it be easier to keep in the compose file?

Reply:
We discussed this and we should look at it again when refactoring the compose file

Reply:
We discussed this and we should look at it again when refactoring the compose file